### ISSUE TITLE
WebNotifications section on html5rocks.com doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#18]
```
We just get an empty notification. Here's the code:

var chart_url = 'html5rocks.com';


document.getElementById('show_html_notification').addEventListener('click', 
function() {
  if (window.webkitNotifications.checkPermission() == 0) {
    // you can pass any url as a parameter
    // note the show()
    window.webkitNotifications.createHTMLNotification(chart_url).show(); 


We're passing the wrong URL there.
```

Original issue reported on code.google.com by `atwilson@chromium.org` on 23 Jun 2010 at 12:42

**Comment by GoogleCodeExporter** [comment]
```
Sorry, can you link to the page?
```

Original comment by `ericbide...@html5rocks.com` on 9 Jul 2010 at 12:04

**Comment by GoogleCodeExporter** [comment]
```
http://playground.html5rocks.com/#web_notifications
```

Original comment by `atwil...@google.com` on 9 Jul 2010 at 12:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `erne...@google.com` on 5 Aug 2010 at 10:47
- Changed state: **Accepted**
- Added labels: **Milestone-3**, **SampleCode**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 05291f629a.
```

Original comment by `erne...@google.com` on 13 Sep 2010 at 7:53
- Changed state: **Fixed**
